### PR TITLE
merge Yosi's team to master

### PR DESCRIPTION
**Pull request by YosWebiks**

### CODE REVIEW

File: IronDome/wwwroot/css/site.css [hunk 1]
```diff
@@ -19,4 +19,8 @@ html {
 
 body {
   margin-bottom: 60px;
+}
+
```
Comment (MikiLevi) [inline IronDome/wwwroot/css/site.css:23]: extra line, please remove

File: IronDome/wwwroot/css/site.css [hunk 2]
```diff
@@ -19,4 +19,8 @@ html {
...
+.yosi{
```
Comment (MikiLevi) [inline IronDome/wwwroot/css/site.css]: try avoidind your name as a class-name

**Review by MikiLevi — CHANGES_REQUESTED** [review]

File: IronDome/wwwroot/css/site.css (hunk 2 above)
Comment (YosWebiks) [reply IronDome/wwwroot/css/site.css]: my name is holy bablical name so i can put it wherever i want

**Review by MikiLevi — APPROVED** [review]

**Comment by lplp9566** [comment]
mikiiiii

**Pull request merged**